servlet/6.0/_index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Jakarta Servlet 6.0 (under development)"
+date: 2021-10-15
+summary: "Release for Jakarta EE 10"
+---
+Jakarta Servlet defines a server-side API for handling HTTP requests and responses.
+
+* [Jakarta Servlet 6.0 Release Record](https://projects.eclipse.org/projects/ee4j.servlet/releases/6.0)
```

Comment:
This link looks correct, but the actual release record has a version of 6.0.0 instead of 6.0.  Specifications should only have a two digit version (major.minor).  So, like I mentioned, the link looks correct, but the actual release record should be updated to be 6.0 instead of 6.0.0.